alembic/versions/bd57477f19a2_add_seen_tables.py

```diff
@@ -0,0 +1,56 @@
+"""add seen tables
+
+Revision ID: bd57477f19a2
+Revises: a4bf1f58ce69
+Create Date: 2020-10-20 22:43:46.743035
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'bd57477f19a2'
+down_revision = 'a4bf1f58ce69'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('seen_files',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('file_id', sa.Integer(), nullable=False),
+    sa.Column('journalist_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['file_id'], ['files.id'], name=op.f('fk_seen_files_file_id_files')),
+    sa.ForeignKeyConstraint(['journalist_id'], ['users.id'], name=op.f('fk_seen_files_journalist_id_users')),
```

Comment:
yup! once https://github.com/freedomofpress/securedrop/issues/5503 is fixed we can expect that journalist_id will never be null and then migrate the table so that both the foreign keys make up the primary key (and remove the unnecessary id column and migrate the null journalist_ids to the id of the deleted user account). until then, journalist_id needs to be nullable.